Packs/CiscoEmailSecurity/Integrations/CiscoEmailSecurity/CiscoEmailSecurity.py

```diff
@@ -0,0 +1,654 @@
+    human_readable = quarantine_message_details_data_to_human_readable(quarantine_message_details.get('attributes'))
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='CiscoEmailSecurity.QuarantineMessageDetail',
+        outputs_key_field='mid',
+        outputs=quarantine_message_details
+    )
+
+
+def list_delete_quarantine_messages_command(client, args):
+    messages_ids = message_ids_to_list_of_integers(args)
+    request_body = {
+        "quarantineType": "spam",
+        "mids": messages_ids
+    }
+    delete_quarantine_messages_response = client.list_delete_quarantine_messages(request_body)
+    total_count = dict_safe_get(delete_quarantine_messages_response, ['data', 'totalCount'], None)
+    return CommandResults(
+        readable_output=f'{total_count} messages successfully deleted from quarantine list',
+    )
+
+
+def list_release_quarantine_messages_command(client, args):
+    messages_ids = message_ids_to_list_of_integers(args)
+    request_body = {
+        "action": "release",
+        "quarantineType": "spam",
+        "mids": messages_ids
+    }
+    release_quarantine_messages_response = client.list_release_quarantine_messages(request_body)
+    total_count = dict_safe_get(release_quarantine_messages_response, ['data', 'totalCount'], None)
+    return CommandResults(
+        readable_output=f'{total_count} messages successfully released from quarantine list',
+    )
+
+
+def build_url_filter_for_get_list_entries(args):
+    limit = set_limit(args.get('limit'))
+    offset = int(args.get('offset', '0'))
+    view_by = args.get('view_by')
+    order_by = args.get('order_by')
+    url_params = f"?action=view&limit={limit}&offset={offset}&quarantineType=spam&orderDir=desc&viewBy={view_by}" \
+                 f"&orderBy={order_by}"
+    return url_params
+
+
+def list_entries_get_command(client, args):
+    list_type = args.get('list_type')
+    url_params = build_url_filter_for_get_list_entries(args)
+    list_entries_response = client.list_entries_get(url_params, list_type)
+    list_entries = list_entries_response.get('data', [None])
+    output_prefix = list_type.title()
+    return CommandResults(
+        readable_output=list_entries,
+        outputs_prefix=f'CiscoEmailSecurity.ListEntry.{output_prefix}',
+        outputs_key_field=output_prefix,
+        outputs=list_entries
+    )
+
+
+def build_request_body_for_add_list_entries(args):
+    request_body = {
+        "action": args.get('action'),
+        "quarantineType": "spam",
+        "viewBy": args.get('view_by')
+    }
+
+    if args.get('recipient_addresses'):
+        request_body["recipientAddresses"] = args.get('recipient_addresses').split(',')
+    if args.get('recipient_list'):
+        request_body["recipientList"] = args.get('recipient_list').split(',')
+    if args.get('sender_addresses'):
+        request_body["senderAddresses"] = args.get('sender_addresses').split(',')
+    if args.get('sender_list'):
```

Comment:
those checks (if statement conditions) should be:
if 'recipient_list' in args
Shouldn't it?

Reply:
You right, done.